kbs/Makefile

```diff
@@ -81,7 +81,7 @@ uninstall:
 	rm -rf $(INSTALL_DESTDIR)/kbs $(INSTALL_DESTDIR)/kbs-client $(INSTALL_DESTDIR)/issuer-kbs $(INSTALL_DESTDIR)/resource-kbs
 
 check:
-	cargo test -p kbs -p kbs-client
+	cargo test -p kbs -p kbs-client -p integration-tests
```

Comment:
This change will get hit [here](https://github.com/confidential-containers/trustee/blob/714b653f3253cb17718df56da9165b86503c0861/.github/workflows/kbs-rust.yml#L81) in our regular workflows, right? (If so, that's what we want, I think.)

Reply:
Yeah. I considered adding another workflow for the integration tests, but I think it's best to keep it together. Note that this is just for checking the code itself, the actual tests will get run via the existing call to `cargo test`